## 83% of end-end tests have passed
```
Total executed tests: 23
Failed tests: 4
Expected to fail tests: 0
Passed tests: 19
```
```
IDs of failed tests: 740490,703391,702839,702731 
```
<h3>Failed tests (4)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/last-login&PR_ID=22450&APK_NAME=22450.apk&TR_CASE_IDS=740490,703391,702839,702731">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832305">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for a message by text: delete for me </div><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'delete for me')]` </div></blockquote></p>

```
communities/test_community.py:409: in test_community_message_delete
    self.channel_2.delete_message_in_chat(message_to_delete_for_me, everyone=False)
../views/chat_view.py:669: in delete_message_in_chat
    self.chat_element_by_text(message).message_body.long_press_without_release()
../views/base_element.py:330: in long_press_without_release
    action.click_and_hold(self.find_element()).perform()
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/common/action_chains.py:103: in perform
    self.w3c_actions.perform()
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/common/actions/action_builder.py:96: in perform
    self.driver.execute(Command.W3C_ACTIONS, enc)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 androidx.test.uiautomator.StaleObjectException; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: androidx.test.uiautomator.StaleObjectException
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:63)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:77)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012)
E   Caused by: androidx.test.uiautomator.StaleObjectException
E   	at androidx.test.uiautomator.UiObject2.getAccessibilityNodeInfo(UiObject2.java:647)
E   	at androidx.test.uiautomator.UiObject2.hashCode(UiObject2.java:105)
E   	at java.lang.Object.toString(Object.java:299)
E   	at java.util.Formatter$FormatSpecifier.printString(Formatter.java:3138)
E   	at java.util.Formatter$FormatSpecifier.print(Formatter.java:3015)
E   	at java.util.Formatter.format(Formatter.java:2678)
E   	at java.util.Formatter.format(Formatter.java:2614)
E   	at java.lang.String.format(String.java:4017)
E   	at io.appium.uiautomator2.utils.ReflectionUtils.invoke(ReflectionUtils.java:85)
E   	at io.appium.uiautomator2.core.AxNodeInfoExtractor.extractAxNodeInfo(AxNodeInfoExtractor.java:58)
E   	at io.appium.uiautomator2.core.AxNodeInfoExtractor.toAxNodeInfo(AxNodeInfoExtractor.java:48)
E   	at io.appium.uiautomator2.model.BaseElement.getBounds(BaseElement.java:171)
E   	at io.appium.uiautomator2.utils.w3c.ActionsTokenizer.extractElementCoordinates(ActionsTokenizer.java:215)
E   	at io.appium.uiautomator2.utils.w3c.ActionsTokenizer.extractCoordinates(ActionsTokenizer.java:307)
E   	at io.appium.uiautomator2.utils.w3c.ActionsTokenizer.extractCoordinates(ActionsTokenizer.java:260)
E   	at io.appium.uiautomator2.utils.w3c.ActionsTokenizer.applyPointerActionToEventsMapping(ActionsTokenizer.java:452)
E   	at io.appium.uiautomator2.utils.w3c.ActionsTokenizer.tokenize(ActionsTokenizer.java:574)
E   	at io.appium.uiautomator2.handler.W3CActions.safeHandle(W3CActions.java:78)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	... 33 more
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9MxFa63PqThAKVQb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpdIyWPfkvSqKpLeC">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832312">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `network-dropdown` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
wallet/test_wallet_mainnet_no_send.py:70: in test_wallet_balance_mainnet
    self.errors.verify_no_errors()
base_test_case.py:205: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: For the Uniswap the wrong value 0.63 is shown, expected 0.627 in total
E    Device 1: For the Uniswap the wrong value 0.13 is shown, expected 0.127 on Mainnet
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESEZo5iYH03A9vaj3h">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832308">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `image-0` </div><div>Device 1: Click system back button </div></blockquote></p>

```
../views/chat_view.py:751: in send_images_with_description
    [self.get_image_by_index(i).click() for i in indexes]
../views/chat_view.py:751: in <listcomp>
    [self.get_image_by_index(i).click() for i in indexes]
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `image-0` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception

During handling of the above exception, another exception occurred:
private_chats/test_one_to_one_chat.py:431: in test_1_1_chat_send_image_save_and_share
    self.chat_1.send_images_with_description(description=image_description, indexes=[0])
../views/chat_view.py:754: in send_images_with_description
    pytest.fail("Can't send image(s) with index(es) %s" % indexes)
E   Failed: Can't send image(s) with index(es) [0]
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyuUqRj94v0HdLEij">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESANuCsAv63mTJEZ6h">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832301">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a pinned message by text: Message 4 </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pinned-messages-menu']//*[starts-with(@text,'Message 4')]` </div></blockquote></p>

```
private_chats/test_one_to_one_chat.py:267: in test_1_1_chat_pin_messages
    pinned_message.long_press_without_release()
../views/base_element.py:330: in long_press_without_release
    action.click_and_hold(self.find_element()).perform()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@content-desc='pinned-messages-menu']//*[starts-with(@text,'Message 4')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyuUqRj94v0HdLEij">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESANuCsAv63mTJEZ6h">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (19)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832320">test_profile_change_profile_photo</a>, id: 741969 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESq3saMeVPHksIax2N">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCCKF0bCWmhnQ2BZr">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832319">test_profile_change_username</a>, id: 741968 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESq3saMeVPHksIax2N">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCCKF0bCWmhnQ2BZr">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832314">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESEZo5iYH03A9vaj3h">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832315">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESEZo5iYH03A9vaj3h">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832313">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESEZo5iYH03A9vaj3h">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832309">test_send_eth[Status Network Sepolia-0.0002]</a>, id: 727229 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES66JQ2LA4SBO3qvpQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIFfRvI2c1xJFBcfV">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832310">test_wallet_send_erc20_from_drawer[Optimism Sepolia-USD Coin-USDC-2-0.01]</a>, id: 727230 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES66JQ2LA4SBO3qvpQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIFfRvI2c1xJFBcfV">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832321">test_send_eth[Arbitrum Sepolia-0.0001]</a>, id: 742015 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES66JQ2LA4SBO3qvpQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIFfRvI2c1xJFBcfV">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832322">test_wallet_send_erc20_from_drawer[Sepolia-USD Coin-USDC-2-0.01]</a>, id: 742016 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES66JQ2LA4SBO3qvpQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIFfRvI2c1xJFBcfV">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832303">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESovjEqqAAfw9jV9Ev">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESDjcUWSDO2u0zhIZn">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESBtfNIH8fuHEA4LCn">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832300">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyuUqRj94v0HdLEij">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESANuCsAv63mTJEZ6h">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832304">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9MxFa63PqThAKVQb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpdIyWPfkvSqKpLeC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832307">test_community_one_image_send_reply_set_reaction</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9MxFa63PqThAKVQb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpdIyWPfkvSqKpLeC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832306">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9MxFa63PqThAKVQb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpdIyWPfkvSqKpLeC">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832302">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5EIJ7AuGD9SSc8Ad">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832317">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9Vqf8Nz4u1l2DBYl">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832316">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9Vqf8Nz4u1l2DBYl">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832318">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9Vqf8Nz4u1l2DBYl">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDeviceTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3832311">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESv7EiexZJwskieXL2">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/profile/data_store.cljs
+++ b/src/status_im/contexts/profile/data_store.cljs
@@ -3,3 +3,7 @@
 (defn accepted-terms?
   [accounts]
   (some :hasAcceptedTerms accounts))
+
+(defn recently-opened-profile
+  [profiles]
+  (first (sort-by :timestamp > (vals profiles))))

--- a/src/status_im/contexts/profile/events.cljs
+++ b/src/status_im/contexts/profile/events.cljs
@@ -79,23 +79,21 @@
       [{accounts                               :accounts
         {:keys [userConfirmed enabled userID]} :centralizedMetricsInfo}
        {:keys [logout-phase?]}]]
-   (let [profiles                (reduce-profiles accounts)
-         profiles-key-uids       (keys profiles)
-         recently-opened-profile (when-not logout-phase?
-                                   (:key-uid (first (sort-by :timestamp > (vals profiles)))))
-         new-db                  (cond-> db
-                                   :always
-                                   (assoc :centralized-metrics/user-confirmed? userConfirmed
-                                          :centralized-metrics/enabled?        enabled
-                                          :centralized-metrics/user-id         userID)
+   (let [profiles          (reduce-profiles accounts)
+         profiles-key-uids (keys profiles)
+         new-db            (cond-> db
+                             :always
+                             (assoc :centralized-metrics/user-confirmed? userConfirmed
+                                    :centralized-metrics/enabled?        enabled
+                                    :centralized-metrics/user-id         userID)
 
-                                   (seq profiles)
-                                   (assoc :profile/profiles-overview profiles))]
+                             (seq profiles)
+                             (assoc :profile/profiles-overview profiles))]
      {:db new-db
       :fx [(when-not logout-phase?
-             [:dispatch [:profile/profile-selected recently-opened-profile]])
-           (when-not logout-phase?
-             [:dispatch [:profile.login/login-with-biometric-if-available recently-opened-profile]])
+             [:dispatch
+              [:profile.login/select-profile-and-login-with-biometric-if-available
+               (-> profiles profile.data-store/recently-opened-profile :key-uid)]])
            [:dispatch [:profile/get-profiles-auth-method profiles-key-uids]]
            (if (profile.data-store/accepted-terms? accounts)
              [:dispatch

--- a/src/status_im/contexts/profile/events.cljs
+++ b/src/status_im/contexts/profile/events.cljs
@@ -77,21 +77,30 @@
  :profile/get-profiles-overview-success
  (fn [{:keys [db]}
       [{accounts                               :accounts
-        {:keys [userConfirmed enabled userID]} :centralizedMetricsInfo}]]
-   (let [profiles          (reduce-profiles accounts)
-         profiles-key-uids (keys profiles)
-         new-db            (cond-> db
-                             :always
-                             (assoc :centralized-metrics/user-confirmed? userConfirmed
-                                    :centralized-metrics/enabled?        enabled
-                                    :centralized-metrics/user-id         userID)
+        {:keys [userConfirmed enabled userID]} :centralizedMetricsInfo}
+       {:keys [logout-phase?]}]]
+   (let [profiles                (reduce-profiles accounts)
+         profiles-key-uids       (keys profiles)
+         recently-opened-profile (when-not logout-phase?
+                                   (:key-uid (first (sort-by :timestamp > (vals profiles)))))
+         new-db                  (cond-> db
+                                   :always
+                                   (assoc :centralized-metrics/user-confirmed? userConfirmed
+                                          :centralized-metrics/enabled?        enabled
+                                          :centralized-metrics/user-id         userID)
 
-                             (seq profiles)
-                             (assoc :profile/profiles-overview profiles))]
+                                   (seq profiles)
+                                   (assoc :profile/profiles-overview profiles))]
      {:db new-db
-      :fx [[:dispatch [:profile/get-profiles-auth-method profiles-key-uids]]
+      :fx [(when-not logout-phase?
+             [:dispatch [:profile/profile-selected recently-opened-profile]])
+           (when-not logout-phase?
+             [:dispatch [:profile.login/login-with-biometric-if-available recently-opened-profile]])
+           [:dispatch [:profile/get-profiles-auth-method profiles-key-uids]]
            (if (profile.data-store/accepted-terms? accounts)
-             [:dispatch [:update-theme-and-init-root :screen/profile.profiles]]
+             [:dispatch
+              [:update-theme-and-init-root :screen/profile.profiles
+               {:default-screen-profiles? logout-phase?}]]
              [:dispatch [:update-theme-and-init-root :screen/onboarding.intro]])
            ;; dispatch-later makes sure that the logout button subscribed is always disabled
            [:dispatch-later

--- a/src/status_im/contexts/profile/login/events.cljs
+++ b/src/status_im/contexts/profile/login/events.cljs
@@ -178,9 +178,10 @@
            [:dispatch [:profile.login/login-existing-profile settings account]]]})))
 
 (rf/reg-event-fx
- :profile.login/login-with-biometric-if-available
+ :profile.login/select-profile-and-login-with-biometric-if-available
  (fn [_ [key-uid]]
-   {:fx [[:effects.biometric/check-if-available
+   {:fx [[:dispatch [:profile/profile-selected key-uid]]
+         [:effects.biometric/check-if-available
           {:key-uid    key-uid
            :on-success (fn [auth-method]
                          (rf/dispatch

--- a/src/status_im/contexts/profile/logout/events.cljs
+++ b/src/status_im/contexts/profile/logout/events.cljs
@@ -36,7 +36,8 @@
     :fx [[:hide-popover nil]
          [:effects.profile/logout nil]
          [:profile.settings/webview-debug-changed false]
-         [:profile/get-profiles-overview #(rf/dispatch [:profile/get-profiles-overview-success %])]]}))
+         [:profile/get-profiles-overview
+          #(rf/dispatch [:profile/get-profiles-overview-success % {:logout-phase? true}])]]}))
 
 (rf/reg-event-fx
  :profile/logout

--- a/src/status_im/contexts/profile/profiles/view.cljs
+++ b/src/status_im/contexts/profile/profiles/view.cljs
@@ -2,6 +2,7 @@
   (:require
     [clojure.string :as string]
     [native-module.core :as native-module]
+    [quo.context :as quo.context]
     [quo.core :as quo]
     [react-native.core :as rn]
     [react-native.reanimated :as reanimated]
@@ -278,7 +279,8 @@
 
 (defn view
   []
-  (let [[show-profiles? set-show-profiles] (rn/use-state true)
+  (let [{:keys [default-screen-profiles?]} (quo.context/use-screen-params)
+        [show-profiles? set-show-profiles] (rn/use-state default-screen-profiles?)
         show-profiles                      (rn/use-callback #(set-show-profiles true))
         hide-profiles                      (rn/use-callback #(set-show-profiles false))]
     (rn/use-mount #(rf/dispatch [:centralized-metrics/check-user-confirmation]))

--- a/src/status_im/contexts/profile/profiles/view.cljs
+++ b/src/status_im/contexts/profile/profiles/view.cljs
@@ -140,9 +140,9 @@
                                :color           customization-color
                                :profile-picture profile-picture})
       :on-card-press        (fn []
-                              (rf/dispatch-sync [:profile/profile-selected key-uid])
                               (rf/dispatch
-                               [:profile.login/login-with-biometric-if-available key-uid])
+                               [:profile.login/select-profile-and-login-with-biometric-if-available
+                                key-uid])
                               (set-hide-profiles))}]))
 
 (defn- profiles-section

--- a/src/status_im/navigation/events.cljs
+++ b/src/status_im/navigation/events.cljs
@@ -77,14 +77,15 @@
 (rf/defn init-root
   "WARNING: Use `:update-theme-and-init-root` instead. `:init-root` should not be used directly."
   {:events [:init-root]}
-  [{:keys [db]} root-id]
-  {:set-root [root-id (:theme db)]})
+  [{:keys [db]} root-id screen-params]
+  {:db       (all-screens-params db root-id screen-params)
+   :set-root [root-id (:theme db)]})
 
 (rf/defn update-theme-and-init-root
   {:events [:update-theme-and-init-root]}
-  [_ root-id]
+  [_ root-id screen-params]
   {:fx [[:dispatch [:theme/switch {:view-id root-id}]]
-        [:dispatch [:init-root root-id]]]})
+        [:dispatch [:init-root root-id screen-params]]]})
 
 (rf/defn hide-bottom-sheet
   {:events [:hide-bottom-sheet]}

--- a/src/tests/integration_test/standard_auth_test.cljs
+++ b/src/tests/integration_test/standard_auth_test.cljs
@@ -25,7 +25,7 @@
   (rf/reg-event-fx :biometric/authenticate
    (fn [_ [{:keys [on-success]}]] (on-success)))
   (rf/reg-event-fx :profile.login/biometric-success (fn [_]))
-  (rf/dispatch [:profile.login/login-with-biometric-if-available :key-uid-1])
+  (rf/dispatch [:profile.login/select-profile-and-login-with-biometric-if-available :key-uid-1])
   (rf-test/wait-for [:profile.login/biometric-success]
     (on-done)))
 

